imod/mf6/out/dis.py

```diff
@@ -114,7 +115,13 @@ def read_hds_timestep(
     return _to_nan(a3d, dry_nan)
 
 
-def open_hds(path: FilePath, d: Dict[str, Any], dry_nan: bool) -> xr.DataArray:
+def open_hds(
+    path: FilePath,
+    d: Dict[str, Any],
```

Comment:
I know this is not your code but could you give 'd' a more descriptive name?
Same applies for the disu and div classes

Reply:
done